### ISSUE TITLE
vignette youtube mélangé

### ISSUE THREAD

**Issue by Nono-m0le** [tontof/kriss_feed#402]
Bon, je sais pas si c'est un bug sur mon instance ou pas, mais de temps en temps, mes vignettes youtube son mélangé.
Exemple, j'ai 3/4 articles à la suite de chaine youtube, dans le 1er article, j'ai la vignette youtube de l'article suivant ...

**Comment by Nono-m0le** [comment]
C'est un peu tout, mais pas forcement que des feeds de youtube, exemple ici : https://i.imgur.com/GF5Bahc.png le contenu de l'article est correct, mais juste la vignette youtube dans l'article, c'est la vidéo d'un autre article (plus bas ou plus haut) qui s'affiche.

en general, je rafraichie, et c'est de nouveau bon.

**Comment by tontof** [comment]
Aucune idée, désolé. En regardant le code source du flux c'est une vidéo directement dans le flux (ce n'est pas une modif de KrISS feed qui incorpore la vidéo), du coup, est-ce que ça peut être lié à un plugin du navigateur ?
Si ça te le fait très souvent, ça vaudrait le coup de tester avec un autre pour voir si ça le fait aussi, genre entre Firefox et Chromium par ex...

**Comment by Nono-m0le** [comment]
J'utilise toujours Firefox, que ce soit sous Linux / Windows / Android.

Comme ca m'arrive aussi sur mobile, et que j'ai que uBlock ... c'est peut-être lié (mais je ne vais pas tester de le désactiver ;-) ).

Par contre, je dois pas être le seul à avoir ce plugin, donc j'ai un doute :s

Qu'est ce qui pourrait t'aider de mon coté ?

3 mai 2018 10:17 "tontof" <notifications@github.com (mailto:%22tontof%22%20<notifications@github.com>)> a écrit:
	Aucune idée, désolé. En regardant le code source du flux c'est une vidéo directement dans le flux (ce n'est pas une modif de KrISS feed qui incorpore la vidéo), du coup, est-ce que ça peut être lié à un plugin du navigateur ?
Si ça te le fait très souvent, ça vaudrait le coup de tester avec un autre pour voir si ça le fait aussi, genre entre Firefox et Chromium par ex...

**Comment by tontof** [comment]
Je ne sais pas vraiment ce qui pourrait m'aider étant donné que je n'ai aucune idée de la source du problème.
Pour info, j'ai aussi uBlock et je n'ai pas le soucis, donc ce n'est sûrement pas ça :-)
Ça te le fait souvent et partout Linux / Windows / Android ?
Tu n'as pas remarqué si les articles que tu avais lu avant de voir le problème venaient d'un flux spécifique ?

**Comment by Nono-m0le** [comment]
J'ai toujours 10 articles chargé, et comme j'ai +100 sources ... c'est difficile et ca peut être un peu tout et n'importe quoi.

En general, ca arrive quand je raffraichie à un endroit, et que la page etait déjà chargé ailleurs.

Je fais mes refresh dans le browser (et pas par cron).

Du coup exemple :
Firefox Android : J'ouvre le matin, je refresh, regarde quelques flux, ca reste ouvert dans firefox, sans que je regarde.
Firefox Windows : J'ouvre le matin je rafraichie, lis des flux etc ...
Pause midi : Je regarde de nouveau sur Firefox android : La, les flux sont "mélangé", il faut que je fasse un refresh manuel pour remettre tout en ordre.

3 mai 2018 10:40 "tontof" <notifications@github.com (mailto:%22tontof%22%20<notifications@github.com>)> a écrit:
	Je ne sais pas vraiment ce qui pourrait m'aider étant donné que je n'ai aucune idée de la source du problème.
Pour info, j'ai aussi uBlock et je n'ai pas le soucis, donc ce n'est sûrement pas ça :-)
Ça te le fait souvent et partout Linux / Windows / Android ?
Tu n'as pas remarqué si les articles que tu avais lu avant de voir le problème venaient d'un flux spécifique ?

**Comment by tontof** [comment]
Effectivement, je n'ouvre jamais KrISS feed à différent endroit en même temps. Il faudrait essayer de le regarder uniquement à un seul endroit pendant un moment pour voir si ça le fait et on pourrait s'orienter vers ça, sachant que je ne vois quand même pas trop comment ça peut influencer ça.

**Comment by Nono-m0le** [comment]
Est-il possible que le cache navigateur ne soit pas geré de la même facon que le cache serveur ?

Du coup, mon cache serveur serait toujours à jour, quand le cache du navigateur lui, à une indexation différente ? (Genre contenu de l'article n° XXX1; dans la div de l'article 01 ?!)

3 mai 2018 10:49 "tontof" <notifications@github.com (mailto:%22tontof%22%20<notifications@github.com>)> a écrit:
	Effectivement, je n'ouvre jamais KrISS feed à différent endroit en même temps. Il faudrait essayer de le regarder uniquement à un seul endroit pendant un moment pour voir si ça le fait et on pourrait s'orienter vers ça, sachant que je ne vois quand même pas trop comment ça peut influencer ça.

**Comment by tontof** [comment]
Une chose est sûre, le nombre d'articles n'est pas correct quand tu ouvres KrISS feed sur plusieurs onglets parce que ce n'est pas géré. Mais ça a normalement aucune incidence sur le contenu des articles, sachant que pour cette vidéo youtube, il s'agit d'une iframe incrustée dans le flux et ça c'est bien récupéré du serveur quand tu le demandes et donc je n'explique pas comment un aperçu de cette iframe peut être différent sachant que de mon point de vue KrISS feed n'a rien à faire là dedans. Lui il fournit juste le code associé à l'iframe et c'est le navigateur qui la charge et affiche l'aperçu... Du coup, je suis un peu perdu

**Comment by Nono-m0le** [comment]
Autre exemple typique (avant de fermer l'issue?) : https://i.imgur.com/gkbw0rZ.png

**Comment by tontof** [comment]
Il y a effectivement quelque chose, mais quoi...
Je viens d'essayer d'afficher 10 par page pour voir si ça m'arrive et si je peux chercher à comprendre d'où ça peut venir

**Comment by tontof** [comment]
Même avec 10 articles par page, je n'ai jamais eu de soucis sur mon instance, je ne sais pas si tu as toujours le problème mais je ferme en attendant d'avoir d'autres retours/pistes.

**Comment by Nono-m0le** [comment]
Je confirme, j'ai toujours le cas. C'est je crois lié à la lecture depuis plusieurs devices..

**Comment by tontof** [comment]
Je n'arrive vraiment pas à comprendre comment cela peut se produire. Je rouvre si par hasard une autre personne à une superbe idée...
